homeassistant/components/pyload/sensor.py

```diff
@@ -132,20 +131,24 @@ async def async_update(self) -> None:
             _LOGGER.info("Authentication failed, trying to reauthenticate")
             try:
                 await self.api.login()
-            except InvalidAuth as e:
-                raise PlatformNotReady(
-                    f"Authentication failed for {self.api.username}, check your login credentials"
-                ) from e
+            except InvalidAuth:
```

Comment:
As a follow up, this should trigger re-authentication :)

Reply:
check 👍

Reply:
had a look at it and a re-authentication doesn't make sense here, because we are already doing a fresh login, we don't have a possibly expired cookie here that needs to be refreshed

Reply:
That sounds odd / wrong. You basically are saying: There is no way for a user to recover from this?

Reply:
If the authentication fails at this point it is because the login credentials are wrong, so currently only changing the login credentials in the configuration.yaml solves this issue, but for future PRs a config flow setup method and a re-auth flow is planned.